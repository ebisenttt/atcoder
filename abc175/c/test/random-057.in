-500824097099959 599325541264625 393602949813313
