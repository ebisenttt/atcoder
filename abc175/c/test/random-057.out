107221147286646
